### PR TITLE
Introduce a VMsize paramter in the runstress function to use custom values for different migration tests. 

### PR DESCRIPTION
**Pull request by shwetaap**
The default VMsize of VMs is less than the VMsize used in the stress command introduced in some of the migration tests. So the stress command never successfully executes in those tests.

Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by vladikr** [comment]
Great finding.
let's wait for the tests to pass.

/approve

**Comment by qinqon** [comment]
"""
/hold
until emergency CI fixes of #5207 are merged
"""

File: tests/migration_test.go
Comment (davidvossel) [inline tests/migration_test.go:321]: should this be `--vm-bytes %sM` ?

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

**Comment by rmohr** [comment]
Seems like a `make` and push of changes is needed.

**Comment by vladikr** [comment]
@rmohr @davidvossel updated.

**Comment by rmohr** [comment]
Waiting for the current running tests to finish before manually merging: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5165/pull-kubevirt-e2e-k8s-1.18/1372107639554576384 and https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5165/pull-kubevirt-e2e-k8s-1.19/1372097446351474688

**Comment by rmohr** [comment]
Tests show clearly no regression. Manually merging to improve CI stability.

**Pull request merged**
